### PR TITLE
FISH-660 Remove Community Profile

### PR DESCRIPTION
**Pull request by MattGill98**
No idea why this is here, it messes with the TCK runners by opaquely changing TCK versions. This is what Git tags and branches are for right?

### CODE REVIEW

**Review by cubastanley — APPROVED** [review]
Please remove the enterprise profile as well, it isn't needed

File: MicroProfile-Metrics/tck-runner/pom.xml [hunk 1]
```diff
@@ -169,10 +151,6 @@
                 </property>
             </activation>
             <properties>
-                <!-- Metrics Dependencies -->
-                <microprofile.metrics.version>2.3</microprofile.metrics.version>
-                <microprofile.metrics.tck.version>2.3</microprofile.metrics.tck.version>
-
                 <!-- Other Test Dependencies -->
                 <cdi.version>2.0.SP1</cdi.version>
```
Comment (cubastanley) [inline MicroProfile-Metrics/tck-runner/pom.xml:155]: Can remove this too and just update the default property to 2.0.SP1

**Review by cubastanley — COMMENTED** [review]
Just one last lil bit

File: MicroProfile-Metrics/tck-runner/pom.xml (hunk 1 above)
Comment (MattGill98) [reply MicroProfile-Metrics/tck-runner/pom.xml:155]: Can we? Community has 2.0 and Enterprise has 2.0.SP1. I'm not sure if that's deliberate? It seemed out of the scope of this issue

**Review by cubastanley — APPROVED** [review]

**Pull request merged**